src/utilities/format-data.js

```diff
@@ -12,3 +12,34 @@ export const changeInsideTagToArray = (body, targetTag, subTag) => (
   x(body, targetTag, [[subTag]])
 );
 
+export function changeDataTableToArray(html){
+  let data = [];
+  x(html, 'tr', [['td']])((err, d) => {
+    data = d;
+  });
+  const year = data[0][0].replace('ปี ', '');
+  data.forEach(e => e.reverse());
+  data.shift();
```

Comment:
It's can work but I think it's harder to understand what is the propose of .
By the way, I will change it.